tox.ini

```diff
@@ -22,7 +22,8 @@ commands =
 
 [flake8]
 max-line-length = 90
-extend-ignore = S101
+extend-ignore = S101, D101, D102, D103, D105, D106, D107, TC006
+per-file-ignores = flake8_trio/visitors/__init__.py: F401, E402
```

Comment:
D101 to D107 is for flake8-docstrings for functions, classes, etc etc. If you want I could enable some of these and get to documenting stuff better.
TC006 is for `typing.cast` where the type isn't given as a string, I didn't see any big reason to bother with that so I ignored it.
`visitors/__init__.py` has the very non-standard imports to populate `ERROR_CLASSES` and `default_disabled_decorators` and noqa'ing each line was fugly, so added them here.

Reply:
I think enforcing per-file docstrings is good when we're starting to hit 20 files - and me documenting stuff should make reviewing easier too~